tedge/src/cli/connect/error.rs

```diff
@@ -38,4 +38,10 @@ pub enum ConnectError {
 
     #[error(transparent)]
     ConfigLoadError(#[from] tedge_config::TEdgeConfigError),
+
+    #[error("Connection check failed")]
+    ConnectionCheckError,
+
+    #[error("Device is not connected to {cloud} cloud")]
+    BridgeFileDoesNotExist { cloud: String },
```

Comment:
Please remove this error. This file is not provided by user, but supposed to be created by the `tedge connect` command.

Reply:
I feel its necessary, I renamed it to "DeviceNotConnected"